api/StorageManager.json

```diff
@@ -13,23 +13,9 @@
           "edge": {
             "version_added": "≤79"
           },
-          "firefox": [
-            {
-              "version_added": "57"
-            },
-            {
-              "version_added": "51",
-              "version_removed": "57",
-              "notes": "See <a href='https://bugzil.la/1304966'>bug 1304966</a> and <a href='https://bugzil.la/1399038'>bug 1399038</a>.",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.storageManager.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            }
-          ],
+          "firefox": {
+            "version_added": "57"
+          },
           "firefox_android": {
             "version_added": "51",
             "notes": "See <a href='https://bugzil.la/1304966'>bug 1304966</a> and <a href='https://bugzil.la/1399038'>bug 1399038</a>.",
```

Comment:
These bugs are the same ones as you removed above, and are closed. Was this really not shipped on Android? If it isn't, another bug needs to be linked.

Reply:
This is shown as supported in Firefox Android, and since the same bugs as Firefox Desktop are linked, it was most likely enabled at the same time as well.  Fixing up the data now!